src/frontend/app/features/applications/application/application-tabs-base/application-tabs-base.component.html

```diff
@@ -19,6 +19,9 @@ <h1>{{ application?.app?.entity?.name }} </h1>
           <mat-icon>play_arrow</mat-icon>
         </button>
       </span>
+      <button mat-icon-button name="cku" routerLink="/applications/{{applicationService.cfGuid}}/{{applicationService.appGuid}}/cli" matTooltip="CLI Info">
+        <mat-icon>help</mat-icon>
```

Comment:
This might get confused with general console/app help. Had a quick look through the other icons, there were some others that might be worth considering?
`code`, `subject`, `short text`, `keyboard` and `power input`,

Reply:
I think only keyboard is a reasonable match